src/rsz/src/Resizer.cc

```diff
@@ -666,25 +726,107 @@ Instance* Resizer::bufferInput(const Pin* top_pin, LibertyCell* buffer_cell)
   string buffer_name = makeUniqueInstName("input");
   Instance* parent = db_network_->topInstance();
   Net* buffer_out = makeUniqueNet();
+  dbNet* buffer_out_net = db_network_->flatNet(buffer_out);
+
   Point pin_loc = db_network_->location(top_pin);
   Instance* buffer
       = makeBuffer(buffer_cell, buffer_name.c_str(), parent, pin_loc);
   inserted_buffer_count_++;
 
+  /*
+    Get buffer output iterm
+   */
+
+  Pin* buffer_ip_pin = nullptr;
+  Pin* buffer_op_pin = nullptr;
+  odb::dbITerm* buffer_ip_iterm;
+
+  odb::dbITerm* buffer_op_iterm;
+  odb::dbBTerm* buffer_bterm_ignore;
+  odb::dbModITerm* buffer_moditerm_ignore;
+  odb::dbModBTerm* buffer_modbterm_ignore;
+
+  getBufferPins(buffer, buffer_ip_pin, buffer_op_pin);
+  db_network_->staToDb(buffer_ip_pin,
+                       buffer_ip_iterm,
+                       buffer_bterm_ignore,
+                       buffer_moditerm_ignore,
+                       buffer_modbterm_ignore);
+  db_network_->staToDb(buffer_op_pin,
+                       buffer_op_iterm,
+                       buffer_bterm_ignore,
+                       buffer_moditerm_ignore,
+                       buffer_modbterm_ignore);
+
   pin_iter = network_->connectedPinIterator(input_net);
   while (pin_iter->hasNext()) {
     const Pin* pin = pin_iter->next();
+
+    if (pin == top_pin) {
+    }
+    //
+    // Get destination type.
+    // could be moditerm
+    // could be modbterm
+    // could be iterm
+    // could be bterm
+    //
```

Comment:
purpose?

Reply:
This comment would hopefully explain what the net connected iterator "destination" type could be. 
(Attention to it would help the prior comment about non-buffer/modinst: note that the iterator over the pins covers the following types of destination: moditerm, modbterm, iterm, bterm). Let's leave it in for now so it is clear what the destination types of the connected pins can be. I agree it is verbose.